QSPIFBlockDevice.cpp

```diff
@@ -732,7 +852,8 @@ int QSPIFBlockDevice::_sfdp_set_quad_enabled(uint8_t *basic_param_table_ptr)
     }
 
     // Read Status Register
-    if (QSPI_STATUS_OK == _qspi_send_general_command(read_register_inst, -1, NULL, 0, (char *)status_reg,
+    if (QSPI_STATUS_OK == _qspi_send_general_command(read_register_inst, QSPI_NO_ADDRESS_COMMAND, NULL, 0,
+            (char *)status_reg,
             sr_read_size) ) {  // store received values in status_value
         tr_debug("DEBUG: Reading Status Register Success: value = 0x%x\n", (int)status_reg[0]);
```

Comment:
Should you check here that setting the QE bit actually succeeded, and either retry or abandon using 4x mode if the flash failed to enter QE mode?

Reply:
@stevew817 - hmm.. I think if we fail here its a major problem. I dont want to implement a retry logic here - it will complicate things and most likely won't help. I also wouldn't want to fallback to 1x mode on failure, because if we got so far the device actually supports modes faster than 1x.
I'm thinking maybe to return -1 and fail init all together , what do you think?